### PR TITLE
Improved logging in API and bugfix in error message handling

### PR DESCRIPTION
**Pull request by johannes-b**
This PR has three topics:
- refactors logging in API
- improves Keptn CLI UX
- bug fix in error message handling for `get event` command

### CODE REVIEW

File: cli/cmd/delete_project.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ Example:
 
 		if len(args) != 1 {
 			cmd.SilenceUsage = false
-			return errors.New("Requires PROJECTNAME")
+			return errors.New(`required argument PROJECTNAME not set`)
```
Comment (christian-kreuzberger-dtx) [inline cli/cmd/delete_project.go]: Is it neccessary to use backticks/backquotes here?

File: cli/cmd/create_project.go [hunk 2]
```diff
@@ -47,7 +47,7 @@ Example:
 
 		if len(args) != 1 {
 			cmd.SilenceUsage = false
-			return errors.New("Requires PROJECTNAME")
+			return errors.New(`required argument PROJECTNAME not set`)
```
Comment (christian-kreuzberger-dtx) [inline cli/cmd/create_project.go]: Same question about backticks/backquotes here.

File: cli/cmd/onboard_service.go [hunk 3]
```diff
@@ -39,7 +39,7 @@ Example:
 	Args: func(cmd *cobra.Command, args []string) error {
 		if len(args) != 1 {
 			cmd.SilenceUsage = false
-			return errors.New("Requires SERVICENAME")
+			return errors.New(`required argument SERVICENAME not set`)
```
Comment (christian-kreuzberger-dtx) [inline cli/cmd/onboard_service.go]: Backticks?

File: cli/cmd/delete_project.go (hunk 1 above)
Comment (johannes-b) [reply cli/cmd/delete_project.go]: No, it is not necessary here, but it does not make a difference.

File: cli/cmd/onboard_service.go (hunk 3 above)
Comment (johannes-b) [reply cli/cmd/onboard_service.go]: see above

File: cli/cmd/create_project.go (hunk 2 above)
Comment (johannes-b) [reply cli/cmd/create_project.go]: see above

**Pull request merged**